var/spack/repos/builtin/packages/py-flameprof/package.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyFlameprof(PythonPackage):
+    """Flamegraph generator for python’s cProfile stats."""
```

Comment:
```suggestion
    """Flamegraph generator for python's cProfile stats."""
```
Unicode characters like smart quotes aren't valid in Python 2, hence the unit test failures.